### ISSUE TITLE
[Question] Network access to entire C: drive instead of specific apps on Simplewall – Bug, misconfiguration, or intrusion?

### ISSUE THREAD

**Issue by Krom5477** [henrypp/simplewall#1887]
Hello, I'm facing an issue with Simplewall. It appears that network access is granted to the entire C: drive (`C:\`) and same for D: (`D:\`) instead of a specific application. No individual app is listed, but network access seems to be given to the whole drive. Is this a bug, a misconfiguration, or an intrusion? How can I fix this? Thank you.
---

App version: <3.8.4 64-bit (Unicode)>
Windows version: <Win10 pro 22h2 >

**Comment by henrypp** [comment]
hi, this is just cosmetics bug "c:\\", it not allow whole C: drive internet access, i fix it when i can

**Comment by Krom5477** [comment]
Ok, thank you very much for the response and the must have software!

**Comment by krzemien77** [comment]
Also present on 3.8.2 as well - see enclosed screenshot taken whilst using second profile on this PC (not present on mine).

![Capture](https://github.com/user-attachments/assets/5f16df3e-72b6-44fd-8f1c-3f29751418d6)

Assume it's the same problem as reported by OP.

**Comment by henrypp** [comment]
@Krom5477, @krzemien77 can you reproduce steps to get "C:\" drive on apps list?

**Comment by krzemien77** [comment]
Not really, not readily, as I'm not sure where this rule comes from in the first place.

User's **profile.xml** under ...\AppData\Roaming\Henry++\simplewall contains simply:

<item path="C:\" timestamp="1723278735" is_undeletable="true"/>

However, and for what it's worth: this timestamp refers to the time when I updated Simplewall to v3.8.3 and encountered previously reported here error with Zoom.exe:

https://github.com/henrypp/simplewall/issues/1495

I checked v3.8.4 also recently and having seen the same problem, decided to roll-back to v3.8.2 - which does seem handle Zoom.exe fine.

**Comment by Krom5477** [comment]
![image](https://github.com/user-attachments/assets/036c7861-f748-4bb2-8e10-7bb7d9e2297a)

I also cannot reproduce the problem because I don't know how it appears. All I can say is that an entry on C:\ with network access reappeared last night at 00:34, and I didn't delete any application at that time or this day. I also sometimes have them on D:\ (but not other disk), which is the drive where I install all applications that do not come from Microsoft, except for Simplewall, which is installed in C:\Program Files\simplewall\simplewall.exe

**Comment by WildTbag** [comment]
I encountered this Bug while right-clicking a process in Tab 'Packets log' to add this process to a User rule.
Instantly **all** but one processes in this specific User rule vanished and the infamous C:\ was added to this rule AND to rules where the vanished processes where ticked as well. In **these** rules only C:\ was added.
Btw i could not figure out why the one unrelated process in the (right-click) rule survived the purge, seems arbritrary.
Thank U very much!
